dozer/sources/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""A collection of sources for various FIRST-related news sources"""
+
+from .RSSSources import *
+from .TwitchSource import TwitchSource
+from .RedditSource import RedditSource
+from .AbstractSources import Source, DataBasedSource
+
+sources = [TwitchSource, RedditSource]
+sources += [source for source in RSSSource.__subclasses__() if not source.disabled]
```

Comment:
This ignores `TwitchSource.disabled` and `RedditSource.disabled`.

Reply:
That's intentional. For these larger, data based classes I feel like disabling them in such way (at least at bot startup) shouldn't be as necessary. For these, just remove it from the list above.